lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -134,15 +136,21 @@ bool LLDBMemoryReader::readBytes(swift::remote::RemoteAddress address,
   LLDB_LOGV(log, "[MemoryReader] asked to read {0} bytes at address {1:x}",
             size, address.getAddressData());
 
+  Address addr = Address(address.getAddressData());
```

Comment:
Suggestion:
we could change resolveFromArtificialToLLDBAddress to
```
lldb::Address LLDBMemoryReader::resolveRemoteAddress(uint64_t address) const { }
```
and have it return `lldb::Address(address)` in the error path. Then this whole block would become

`lldb::Address addr = resolveRemoteAddress(address.getAddressData());`